package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
+      "function-parentheses-space-inside": "always",
```

Comment:
We require spaces inside parens for PHP and JavaScript, as such this is more consistent with CSS too.